doc/ref_model/chapters/appendix-a.md

```diff
@@ -16,19 +16,75 @@ This Appendix has two goals:
+- d. Application control using RA APIs finds general profile node without any specific infrastructure component.
+
+**2. Platform as a Service** (as per RA2 WIP: common services starting from Load Balancer and Store State, also typically includes Logging and Telemetry):
+- a. Application functionality or application control can work only with its own components implementing platform services already provided by CNTT-defined PaaS.
+- b. With custom integration effort, application can be made to use CNTT-defined PaaS.
+- c. Application is designed and can be configured for running with CNTT-defined PaaS.
+
+**3. Application Resiliency** (Note: “resilient” is the 1st word describing “enabled loosely coupled systems” in CNCF Cloud Native Definition [6])
+- a. Application was designed and tested to run only on Carrier Grade platform with predictable infrastructure availability and performance
+- b. Application was designed and tested for HW and SW component full failures but not for infrastructure impairment which still needs predictable infrastructure performance (like CPU cycles and network latencies)
+- c. Application was designed to run on shared Cloud platforms and tested for resilience to infrastructure impairments
+
+Relevant for sizing infrastructure and application operations (which often is another telco organizational unit or external 3rd party running on telco cloud) is also how much is application decomposed from:
+
+**4. Other application functionality** (decomposition, manageability):
+- a. Application consists of huge monolithic components including algorithms that have different scaling (for example depending on type of traffic) and/or availability requirements
+- b. Application is decomposed into smaller components that are tightly coupled by same versioning (upgrading all components or none)
+- c. Decomposed application has components that can be upgraded separately
+- d. Availability like N+K or 1:1 is defined during application design and not configurable at deployment time.
+- e. Patching/upgrading existing running application components or always rolling out new ones.
+
+(WIP: After agreeing above criteria, next is defining number and naming of Maturity Levels and mapping the criteria to different levels.)
+
 <a name="A.4"></a>
 ## A.4 Links
 
+1. [Wikipedia:Loose Coupling](https://en.wikipedia.org/wiki/Loose_coupling)
+2. [Wikipedia:Encapsulation](https://en.wikipedia.org/wiki/Encapsulation_(computer_programming))
+3. [Wikipedia:Decomposition](https://en.wikipedia.org/wiki/Decomposition_(computer_science))
+4. [Wikipedia:Resilience](https://en.wikipedia.org/wiki/Resilience_(network))
+5. [Wikipedia:Observability](https://en.wikipedia.org/wiki/Observability)
+
+6. [CNCF Cloud Native Definition v1.0 from June 2018](https://github.com/cncf/toc/blob/master/DEFINITION.md)
+
 <a name="A.5"></a>
 ## A.5 Hardware-Dependent Coding Policies
```

Comment:
@petorre This section has been moved to tech/policy. you might need to add the change you propose to there.

Reply:
Yes, is now in #1367 .